.github/workflows/build.yml

```diff
@@ -38,10 +32,12 @@ jobs:
         with:
           distribution: temurin
           java-version: ${{ env.JAVA_VERSION }}
-          cache: gradle
```

Comment:
Even this line as well?

Reply:
`gradle/actions/setup-gradle` can do this better, we don't need to cache again in `setup-java`.